### DISCUSSION TITLE
[github action ui] cannot select the branch on the phone

### DISCUSSION THREAD

**Discussion by zhangguanzhang** [community/community#17046] [Mobile] [answered]
![72Q2`M4C76OYL 3I_(U9D~H](https://user-images.githubusercontent.com/18641678/169653034-cecf7022-b819-4767-a84f-445b88706645.jpg)
![{M93{RCCG1(YM6V4)ULWZ0M](https://user-images.githubusercontent.com/18641678/169653039-d253288d-4a59-4736-934e-c3b8b7ec5886.jpg)
![YWRVYNZB 6% NX~4AMC%3`Y](https://user-images.githubusercontent.com/18641678/169653044-b7f9c085-c629-4b62-b9cb-06e03ebf0f53.jpg)
![W0C1Q`JAMAV)MUXEK8BG6HL](https://user-images.githubusercontent.com/18641678/169653051-8fe91553-980d-4588-8f3a-d0e27c67e3d3.jpg)
I just want to choose a branch to run, But is not display for me

**Comment by zhangguanzhang** [comment]
Is there someone to look? 🤔

**Comment by zhangguanzhang** [comment]
Is there someone to look? 🤔 2022/06/06

**Comment by ShenMian** [comment]
I think mobile means [Github Mobile](https://github.com/mobile), a mobile app.

**Reply by zhangguanzhang** [reply]
this is not a good solution 😢

**Reply by ShenMian** [reply]
Because it's not a good question.

**Comment by zhangguanzhang** [comment, marked as answer]
Todaty，this problem had been resolved by the developer for the github web. People don't have to download the github app separately
